resource_elasticsearch_xpack_role_mapping_test.go

```diff
@@ -216,3 +216,43 @@ resource "elasticsearch_xpack_role_mapping" "test" {
 }
 `, resourceName)
 }
+
+func TestAccRoleMappingResource_importBasic(t *testing.T) {
+	provider := Provider().(*schema.Provider)
+	err := provider.Configure(&terraform.ResourceConfig{})
+	if err != nil {
+		t.Skipf("err: %s", err)
+	}
+	meta := provider.Meta()
+	var allowed bool
+	switch meta.(type) {
+	case *elastic5.Client:
+		allowed = false
+	default:
+		allowed = true
+	}
+
+	randomName := "test" + acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+
+	resource.Test(t, resource.TestCase{
+                PreCheck: func() {
+			testAccPreCheck(t)
+			if !allowed {
+				t.Skip("Role Mappings only supported on ES >= 6")
+			}
+		},
+		Providers:    testAccXPackProviders,
+		CheckDestroy: testAccCheckRoleMappingDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccRoleMappingResource(randomName),
+			},
+			{
+				ResourceName:      "elasticsearch_xpack_role_mapping.test",
+				ImportState:       true,
+				ImportStateVerify: true,
+                                ImportStateVerifyIgnore: []string{"role_mapping_name"}, // we either found it by name or it's not there
```

Comment:
This spacing seems off, can you `go fmt` these files? Odd that CI didn't catch this.

Reply:
Sure thing! Done in https://github.com/phillbaker/terraform-provider-elasticsearch/pull/58/commits/cc512842f5ffa851137747170a42a768b03a1738